UsingModules/UsingModules/UsingModules/Views/MainNavigationPage.xaml

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<NavigationPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:views="clr-namespace:UsingModules.Views;assembly=UsingModules"
+             x:Class="UsingModules.Views.MainNavigationPage"
+             Title="MainPage">
+  <views:MainPage />
```

Comment:
Since you are navigating in the OnInitialized to `NavigateAsync("MainNavigationPage/MainPage")` you do not need to declare `views:MainPage` in your navigation page.